test/integration/metrics-compliance-static-analysis.py

```
@@ -0,0 +1,547 @@
+            },
+            "RHOAI image adoption": {
+                "metrics": ["rhoai", "external"],
+                "description": "Measure RHOAI vs external images"
+            },
+            "Framework prioritization": {
+                "metrics": ["pytorch", "tensorflow", "other"],
+                "description": "Identify which framework to invest in"
+            }
+        }
+        
+        all_covered = True
```

Ruff (0.12.2)

Local variable `all_covered` is assigned to but never used

Remove assignment to unused variable `all_covered`

(F841)